public/app/components/user/editUser/editUser.dir.js

```diff
@@ -65,6 +65,8 @@
 
 			function saveField(ev, item) {
 				item.saving = true;
+				$scope.profile.date_of_birth =  new Date($scope.profile.date_of_birth.year,$scope.profile.date_of_birth.month.value,$scope.profile.date_of_birth.day);
```

Comment:
Looks like this is on the root scope (Which it shouldn't be) so if it's not change the name to $user or whatever.